src/sentry/replays/blueprints/api.md

```diff
@@ -246,6 +246,10 @@ This resource does not accept any URI parameters and is not paginated. Responses
 
 ### Fetch Replay Accessibility Issues [GET]
 
+- Parameters
+
+  - timestamp (optional, number) - The timestamp which to render to for accessibility analysis.
```

Comment:
should we mention that it's in seconds since the epoch?

the unit (seconds) is really the important bit :D

Reply:
Sounds like a really good idea